CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
-# Notice
+# Aggregate Changelog
 
-**changelogs for each driver are each GitHub repository. Please check them directly. https://appium.io/docs/en/latest/ecosystem/ has links for each driver.**
+Appium contains many different modules and supports a variety of drivers, most of which have their own changelogs. Links to these individual changelogs are aggregated here.
 
-Changelogs in Appium 1.x is in https://github.com/appium/appium/blob/1.x/CHANGELOG.md
+## Main Appium Module
+
+* ### [Appium 2.0 Changelog](packages/appium/CHANGELOG.md)
+* [Appium 1.x Changelog](https://github.com/appium/appium/blob/1.x/CHANGELOG.md)
+
+## Appium Drivers
```

Comment:
maybe this section should be `Drivers and Plugins`?

Reply:
Hmm, all the plugin links are already listed in the Other Modules section, but I do have an idea...